tests/scripts/test_psa_compliance.py

```diff
@@ -50,12 +50,9 @@ def main(library_build_dir: str):
 
     in_tf_psa_crypto_repo = build_tree.looks_like_tf_psa_crypto_root(root_dir)
```

Comment:
This variable is no longer used.

Reply:
Just drawing attention to [this comment](https://github.com/Mbed-TLS/mbedtls/pull/8523#issuecomment-1825547608) for clarity.